File: generator/src/java/darkness/generator/scripts/uka17/CountDown.java
package darkness.generator.scripts.uka17;

public class CountDown extends BaseScript {
    @Override
    public void run() {
        super.run();
        for (int i = 5; i > 1; i--) {
            setCounter(i, true);
            skip(60 * 20);
        }
        for (int i = 60; i >= 0; i--) {
            setCounter(i, false);
            skip(20);
        }
    }
}

File: generator/src/kotlin/darkness/generator/scripts/uka17/CountDown.kt
package darkness.generator.scripts.uka17

class CountDown : BaseScript() {
    override fun run() {
        super.run()
        for (i in 5 downTo 2) {
            setCounter(i, true)
            skip(60 * 20)
        }
        for (i in 60 downTo 0) {
            setCounter(i, false)
            skip(20)
        }
    }
}
